### PR TITLE
Add Sensor, BinarySensor to geniushub

### PR DESCRIPTION
**Pull request by zxdavb**
**Description:**

Adds BinarySensor and Sensor components to the geniushub integration.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#23811

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

**Review by MartinHjelmare — DISMISSED** [review]

**Review by frenck — CHANGES_REQUESTED** [review]
@zxdavb Could you take a look at the merge conflict? As soon as that is resolved, we can go and merge this in. Thanks already! 👍

File: source/_components/geniushub.markdown [hunk 1]
```diff
@@ -47,7 +47,22 @@ Each such entity will report back its primary state; in addition, `assigned_zone
 
 There are three `Sensor` entities that will indicate the number of **Errors**, **Warnings** and **Information** issues.
 
-Each such entity has a state attribute that will contain a list of any such issues. For example, `error_list`.
+Each such entity has a state attribute that will contain a list of any such issues which can be used in automations, etc. For example:
+
+```yaml
```
Comment (frenck) [inline source/_components/geniushub.markdown:53]: This block contains a jinja template and needs to be escaped using `{% raw %}`

Comment (zxdavb) [reply source/_components/geniushub.markdown:53]: Yeah - I dunno why I did that - trying to put it back now, but deploy check is fighting me!

**Review by frenck — APPROVED** [review]
Thanks, @zxdavb! Looks great now! 👍

**Comment by zxdavb** [comment]
Thanks - now just have to get the PR accepted.

**Comment by frenck** [comment]
@zxdavb Uh? home-assistant/home-assistant#23811 has been accepted.... ?

**Comment by klaasnicolaas** [comment]
Uhm did we forgot this PR for the 0.94 release?

**Comment by frenck** [comment]
@klaasnicolaas yep!

**Pull request merged**